ent storage cache (#1189)
vesoft-inc/nebula-docs: docs-2.0/5.configurations-and-logs/1.configurations/4.storage-config.md

For more information, see [RocksDB official documentation](https://rocksdb.org/).

## Storage cache configurations

!!! enterpriseonly

        Only available for the Nebula Graph Enterprise Edition.

| Name                          | Predefined value  | Description                   |
| :-----------------------------| :------| :------------------------------- |

> | `enable_storage_cache`        | `false`|  Whether or not to enable storage cache. |

Reviewer: ```suggestion
| `enable_storage_cache`        | `false`|  Whether or not to cache Storage data. |
```

> | `storage_cache_capacity`      | `0`    |  Total capacity reserved for storage in memory cache. The value must be slightly greater than the sum of `vertex_pool_capacity` and `empty_key_pool_capacity`. The configuration is measured in MB.|

Reviewer: ```suggestion
| `storage_cache_capacity`      | `0`    |  The size of memory reserved for Storage caches. The value must be slightly greater than the sum of `vertex_pool_capacity` and `empty_key_pool_capacity`. The configuration is measured in MB.|
```

> | `storage_cache_buckets_power` | `20`   |  The number of buckets. The value is a logarithm base 2, optional values are 0~32. For example, the value 20 indicates that the number of buckets is 2$^{20}$. The recommended value is `ceil(log2(cacheEntries * 1.6))`. `cacheEntries` indicates the total number of cache items.|

Reviewer: ```suggestion
| `storage_cache_buckets_power` | `20`   |  The number of buckets. The value is a logarithm with a base of 2. Optional values are 0~32. For example, the value 20 indicates that the number of buckets is 2$^{20}$. The recommended value is `ceil(log2(cacheEntries * 1.6))`. `cacheEntries` indicates the total number of cache items.|
```

> | `storage_cache_locks_power`   | `10`    | The number of locks. The value is a logarithm base 2, optional values are 0~32. For example, the value 10 indicates that the number of locks is 2$^{10}$. The recommended value is `max(1, storage_cache_buckets_power - 10)`. |

Reviewer: ```suggestion
| `storage_cache_locks_power`   | `10`    | The number of locks. The value is a logarithm with a base of 2. Optional values are 0~32. For example, the value 10 indicates that the number of locks is 2$^{10}$. The recommended value is `max(1, storage_cache_buckets_power - 10)`. |
```

> | `enable_vertex_pool`          | `false`| Whether or not to add vertex pool in the cache. Only valid when storage cache is enabled.  |

Reviewer: ```suggestion
| `enable_vertex_pool`          | `false`| Whether or not to add a vertex pool in the cache. Only valid when the storage cache feature is enabled.  |
```

> | `vertex_pool_capacity`        | `50`   | The size of vertex pool. The configuration is measured in MB.|

Reviewer: ```suggestion
| `vertex_pool_capacity`        | `50`   | The size of the vertex cache pool. The configuration is measured in MB.|
```

> | `vertex_item_ttl`             | `300`  | The TTL of items of vertex pool. The configuration is measured in seconds. |

Reviewer: ```suggestion
| `vertex_item_ttl`             | `300`  | The TTL of vertex cache pool items. The configuration is measured in seconds. |
```

> | `enable_empty_key_pool`       | `false`| Whether or not to add empty_key pool in the cache. Only valid when storage cache is enabled. The empty_key indicates the queried key but does not exist.  |

Reviewer: ```suggestion
| `enable_empty_key_pool`       | `false`| Whether or not to add an empty_key pool in the cache. Only valid when the storage cache is enabled. The empty_key indicates a key that was queried but does not actually exist.  |
```

> | `empty_key_pool_capacity`     | `50`   | The size of empty_key pool. The configuration is measured in MB. |

Reviewer: ```suggestion
| `empty_key_pool_capacity`     | `50`   | The size of the empty_key cache pool. The configuration is measured in MB. |
```

> | `empty_key_item_ttl`          | `300`  | The TTL of items of empty_key pool. The configuration is measured in seconds.  |

Reviewer: ```suggestion
| `empty_key_item_ttl`          | `300`  | The TTL of the empty_key cache pool items. The configuration is measured in seconds.  |
```
